app/views/admin/dashboard/show.html.erb

```diff
@@ -0,0 +1,22 @@
+<h1>Your role: admin</h1>
+<h3>Users:</h3>
+<table>
+  <tr>
+    <th>ID</th>
+    <th>Name</th>
+    <th>Status</th>
+    <th>Change status</th>
+  </tr>
+  <% @users.each do |user| %>
+  <tr>
+    <td><%= user.id %></td>
+    <td><%= user.name %></td>
+    <td><%= user.status %></td>
+    <% if user.status == "active" %>
```

Comment:
I've seen user.status == "active" a few times in the PR. Maybe this is something we can pull into a #active? method on the User model at some point?